deploy/examples/cluster.yaml

```
@@ -113,10 +113,10 @@ spec:
     # specified number of days.
     #daysToRetain: 30
   # enable log collector, daemons will log on files and rotate
-  # logCollector:
-  #   enabled: true
-  #   periodicity: daily # one of: hourly, daily, weekly, monthly
-  #   maxLogSize:  500M # SUFFIX may be 'M' or 'G'. Must be at least 1M.
+  logCollector:
+    enabled: true
+    periodicity: hourly # one of: hourly, daily, weekly, monthly
```

I think daily makes more sense as a default. Otherwise, logs will be rotated so frequently they will expire before the admin can take a look.
```suggestion
    periodicity: daily # one of: hourly, daily, weekly, monthly
```

---

@travisn sure.